### PR TITLE
Feature navbar

### PR DESCRIPTION
**Pull request by Kelleshia**
Navigation UI added and await review

### CODE REVIEW

File: src/components/layout/index.js [hunk 1]
```diff
@@ -8,8 +8,8 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { useStaticQuery, graphql } from "gatsby"
-
-import Header from "../header"
+import Navbar from "../navbar/navbar"
```
Comment (Natreve) [inline src/components/layout/index.js:11]: you can change this to _**components/navbar**_ and then add a index.js file and move all related components into one file and export them, look at components/list for example

File: src/components/navbar/avatar.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react"
```
Comment (Natreve) [inline src/components/navbar/avatar.js:1]: Move this into the index.js file you will create at components/navbar and export it

File: src/components/navbar/logo.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from "react"
```
Comment (Natreve) [inline src/components/navbar/logo.js:1]: Move this into the index.js file you will create at components/navbar and export it

File: src/components/navbar/navbar.js [hunk 4]
```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react"
```
Comment (Natreve) [inline src/components/navbar/navbar.js:1]: Move this into the index.js file you will create at components/navbar and export it

File: src/components/layout/index.js (hunk 1 above)
Comment (Kelleshia) [reply src/components/layout/index.js:11]: Will do, thanks

**Review by Natreve — CHANGES_REQUESTED** [review]
Change mentioned.

**Pull request closed without merging**